src/SavedSearches/Command/UpdateSavedSearchJSONDeserializer.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\SavedSearches\Command;
+
+use CultuurNet\UDB3\Deserializer\JSONDeserializer;
+use CultuurNet\UDB3\Deserializer\MissingValueException;
+use CultuurNet\UDB3\SavedSearches\Properties\QueryString;
+
+/**
+ * @deprecated
```

Comment:
A bit weird to create new code and already mark it as deprecated?

Reply:
It is deprecated because the new way is with making a json schema, and using that is validation.
But none of these endpoints have documentation because they are internal.
I discussed this with Corneel and decided this is currently out of scope.